WakaBot.Web/Program.cs

```diff
@@ -22,6 +25,22 @@
     return "Up time is " + stopwatch.Elapsed;
 });
 
+app.MapGet("/metrics", () =>
+{
+    StringBuilder sb = new StringBuilder();
+    sb.AppendLine("waka_uptime " + stopwatch.Elapsed);
+    sb.AppendLine("waka_memory_usage " + Process.GetCurrentProcess().PrivateMemorySize64);
+
+    var database = app.Services.GetService<WakaContext>();
```

Comment:
@djpiper28 WakaBot.Web and WakaBot.Core has two different instances of DI. To get this to work you may first need to access WakaBotService via DI, then access the bots services from there.